e2e/src/ui/webElements/modelsDialog.ts

```diff
@@ -106,26 +105,16 @@ export class ModelsDialog extends BaseElement {
     ).getComputedStyleProperty(Styles.textColor);
   }
 
-  public async getEntitiesIconAttributes() {
-    const allIcons: Icons[] = [];
+  public async getEntitiesIcons() {
```

Comment:
This method duplicates `getRecentEntitiesIconAttributes` with difference `this.recentEntities`<->`this.groupEntity` and `ChatSelectors.recentEntityNames`<->`ModelDialog.groupEntityName` which can be used as arguments util-method

Reply:
good catch, thank you!